contrib/database/sql/conn.go

```diff
@@ -151,7 +164,7 @@ func WithSpanTags(ctx context.Context, tags map[string]string) context.Context {
 }
 
 // tryTrace will create a span using the given arguments, but will act as a no-op when err is driver.ErrSkip.
-func (tp *traceParams) tryTrace(ctx context.Context, resource string, query string, startTime time.Time, err error) {
+func (tp *traceParams) tryTrace(ctx context.Context, queryType queryType, query string, startTime time.Time, err error) {
```

Comment:
```suggestion
func (tp *traceParams) tryTrace(ctx context.Context, qtype queryType, query string, startTime time.Time, err error) {
```
Just to reduce confusion :)